Fix some strings not translatable

mrtuvn opened the pull request:
<!---
    Thank you for contributing to Magento.
    To help us process this pull request we recommend that you add the following information:
     - Summary of the pull request,
     - Issue(s) related to the changes made,
     - Manual testing scenarios
    Fields marked with (*) are required. Please don't remove the template.
-->

<!--- Please provide a general summary of the Pull Request in the Title above -->

### Description (*)
<!---
    Please provide a description of the changes proposed in the pull request.
    Letting us know what has changed and why it needed changing will help us validate this pull request.
-->
Init part theme and translate in very first dispatch from FrontController
https://devdocs.magento.com/guides/v2.4/extension-dev-guide/routing.html

One main archieve in this PR is to reduce frequence high load from design plugin `designLoader` in 

https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Theme/etc/di.xml#L109 by move load parts to FrontController instead called in plugin beforeExecute. Only keep on line for check design change

Another approach possible to do is use Observer we listens event `controller_action_predispatch`
load design + translate when event fired


### Related Pull Requests
<!-- related pull request placeholder -->

### Fixed Issues (if relevant)
<!---
    If relevant, please provide a list of fixed issues in the format magento/magento2#<issue_number>.
    There could be 1 or more issues linked here and it will help us find some more information about the reasoning behind this change.
-->
1. Fixes magento/magento2#31849

### Manual testing scenarios (*)
<!---
    Please provide a set of unambiguous steps to test the proposed code change.
    Giving us manual testing scenarios will help with the processing and validation process.
-->
1. Following steps in report issues

### Questions or comments
<!---
	If relevant, here you can ask questions or provide comments on your pull request for the reviewer
	For example if you need assistance with writing tests or would like some feedback on one of your development ideas
-->

CC: @siimm @lbajsarowicz @Vinai @sidolov @gabrieldagama @zakdma 

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

siimm commented on app/code/Magento/Theme/etc/di.xml:
```diff
@@ -105,9 +105,6 @@
         </arguments>
     </virtualType>
-    <type name="Magento\Framework\App\ActionInterface">
-        <plugin name="designLoader" type="Magento\Theme\Plugin\LoadDesignPlugin"/>
```
The class `Magento\Theme\Plugin\LoadDesignPlugin` can probably removed alltogether after this change ?
As I see this was the only use of that class.

mrtuvn replied:
Sure yes you can do that

mrtuvn replied:
In fact we should disable or deprecate this file instead completely remove it for maintain backward compatible

mrtuvn commented:
@magento run all tests

mrtuvn commented:
@magento run all tests

mrtuvn commented:
@magento run Functional Tests B2B

mrtuvn commented:
Seem this PR fix problem reported by you @siimm
https://github.com/magento/magento2/pull/31970/

mrtuvn commented:
@magento run all tests

mrtuvn commented:
Closing this since problem fixed by another pull request mentioned previous above

mrtuvn commented:
@magento run all tests

mrtuvn commented:
@magento run all tests

mrtuvn commented on lib/internal/Magento/Framework/View/DesignLoader.php:
```diff
@@ -51,8 +51,10 @@ public function __construct(
+            $this->_request instanceOf RequestInterface) {
+            $area->detectDesign($this->_request);
+        }
     }
```
@Vinai @lbajsarowicz 
moved load part to FrontController to reduce frequently load to much

mrtuvn commented:
@magento run all tests

mrtuvn commented:
@magento run all tests

mrtuvn commented on lib/internal/Magento/Framework/App/FrontController.php:
```diff
@@ -137,6 +137,9 @@ public function dispatch(RequestInterface $request)
+        $area = $this->areaList->getArea($this->appState->getAreaCode());
+        $area->load(Area::PART_DESIGN);
+        $area->load(Area::PART_TRANSLATE);
         while (!$request->isDispatched() && $routingCycleCounter++ < 100) {
```
The only concern here is at the moment run into this method `dispatch` some reason what if `$area` not available ?

mrtuvn committed:
Fix some strings not translatable


Update files


update


update deprecate note


update note 


Update


update


update


update


update


update


up

mrtuvn commented:
@magento run all tests

mrtuvn commented:
@magento run Functional Tests EE , Functional Tests CE, Functional Tests B2B

mrtuvn commented:
@engcom-Alfa can you check with this change ?

mrtuvn commented:
@magento run Functional Tests EE, Functional Tests B2B

mrtuvn commented:
@magento run Functional Tests EE, Functional Tests B2B

mrtuvn commented:
@magento run Functional Tests EE, Functional Tests B2B

mrtuvn commented:
@magento run Functional Tests EE, Functional Tests B2B

mrtuvn commented:
closed this PR when origin issue can't reproduce
